tests/subsys/logging/log_stack/src/test.inc

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ztest.h>
+#include <logging/log.h>
+
+#define MODULE_NAME test
+#define CXX_MODULE_NAME test_cxx
+
+/* The idea of this test is to estimate stack usage of logging messages. Each
+ * test checks stack usage before and after logging a message. Increase of stack
+ * usage is interpreted as stack used by the logging. Test has been used to
+ * characterize some common architectures and it will fail if more stack is
```

Comment:
```suggestion
 * characterize some common architectures and it will fail if significantly more stack is
```

I assume we don't want to maintain this limit for every byte that is incremented?